common/src/main/java/org/wildfly/httpclient/common/WildflyHttpContext.java

```diff
@@ -44,6 +45,7 @@
 public class WildflyHttpContext implements Contextual<WildflyHttpContext> {
 
     private static final int LEAK_DETECTION = Integer.getInteger("org.wildfly.http-client.buffer-leak-detection", 0);
+    private static final boolean TCP_NODELAY = Boolean.getBoolean("org.wildfly.httpclient.tcp-nodelay");
```

Comment:
Please change the system property name to match the already existing naming scheme to `org.wildfly.http-client.tcp-nodelay`

Reply:
Please change from "org.wildfly.httpclient.tcp-nodelay" to "org.wildfly.httpc-lient.tcp-nodelay" @rachmatowicz

Reply:
Yes, we need to make it consistent with the other properties in the project.